content/en/users/cloud-compute/ec3/apps/_index.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "EC3 applications and tools"
+type: docs
+weight: 30
+description: >
+  How to run scientific applications and tools with EC3
+---
+
+This section documents how to run some applications and use the existing tools
+with EC3.
+
+## How to run scientific applications in EC3
+
+### NAMD cluster
+
+To deploy NAMD clusters, please select one of the available LRMS (Local Resource
```

Comment:
A link/reference/description of NAMD could help people not knowing it figure if it could interest them

Reply:
Maybe we should at least link to project homepages for all the services/tools that are referenced?

Reply:
Agree:

NAMD: https://www.ks.uiuc.edu/Research/namd/
Apache Mesos: http://mesos.apache.org/
Chronos: https://mesos.github.io/chronos/
Kubernetes: https://kubernetes.io/
Marathon: https://mesosphere.github.io/marathon/